### DISCUSSION TITLE
Android WearOS & Capacitor

### DISCUSSION THREAD

**Discussion by tonyawad88** [ionic-team/capacitor#3455] [Ideas]
Hello,

I am wondering if there's a possibility to have Capacitor build/deploy Android Wear OS applications? Is there any plan to include it as part of Capacitor in the future? Maybe with the potential Ionic native elements?

Any thoughts / comments are greatly appreciated.
Thank you!

**Comment by mhartington** [comment]
A far as I am aware, WearOS or any wearable OS do not have a webview that can support this.

**Reply by tonyawad88** [reply]
Agreed. When Max presented at the last conference the Ionic Native Elements where certain ionic components are translated / converted to native elements, is that something possible or on the roadmap for wearos as well?

**Reply by mhartington** [reply]
Not 100% what he meant. For example, let's look at modals. With Capacitor Elements, the goal will be to use native modals/overlays in iOS and Android, but just use a web-based modal when deployed to the browser. It doesn't mean that will be converted to native UI.

**Reply by tonyawad88** [reply]
Oh okay I might have misunderstood what he meant. So you think he meant, Ionic / Capacitor under the hood can invoke a native modal on native devices when a ionic modal is presented. Rather than actually a ui conversion...  

For some reason I got the hint that there's a ui conversion happening somewhere where even a button for example will be converted to a native one and somehow embedded / overlayed into / onto the webview using some Ionic magic. 

Settled for now I guess. No wearos until WearOS gets a webview :)

**Comment by ZanzyTHEbar** [comment]
wearOS has webview now, can capacitor deploy to wearOS now?
